docs/how_to_guides/how_to_process_previously_recorded_videos.md

```diff
@@ -1,54 +1,82 @@
 
 # How to process pre-recorded synchronized videos with `alpha` GUI
 
-Here's a workflow you can use to pre-process videos recorded with the `pre-alpha (<=v0.0.54)` using the `alpha` 
+This is a workflow you can use to process **any** pre-recorded & synchronized videos.
+
+!!! take-note "Note"
+
+    This software is a work-in-progress, you'll likely have to click around and test things out to get this process to work for the first time. Often, you might click a button and want to go back, but the GUI doesn't have "go-back" capabilities yet :) So in order to go back, simply restart the GUI. You can quickly restart the GUI at any time by selecting the GUI and pressing `ctrl-R`
```

Comment:
This is a helpful note, but it doesn't 'scan' well - maybe something like: 

----

This software is a work-in-progress, you'll likely have to click around and test things out to get this process to work for the first time. 

Often, you might click a button and want to go back, but the GUI is missing some "go-back" capabilities. 

If you need to,  **you can restart the GUI at any time by selecting the GUI and pressing `ctrl-R` and then re-load whichever session you were working on with `Ctrl-D`**